pkg/chartutil/create.go

```diff
@@ -147,16 +147,15 @@ service:
 
 ingress:
   enabled: false
+  className:
   annotations: {}
     # kubernetes.io/ingress.class: nginx
     # kubernetes.io/tls-acme: "true"
   hosts:
     - host: chart-example.local
       paths:
-      - path: /
-        backend:
-          serviceName: chart-example.local
-          servicePort: 80
+        - path: /
```

Comment:
indentation seems to be incorrect.

Reply:
Why? What's wrong with it?

Reply:
in most of the case node's children are intended by two characters (see hosts -> host).
on paths -> path it was not; márk fixed that (also on other parts of the file). it looks good for me